develop/docs/dlq.md

```diff
@@ -0,0 +1,35 @@
+# Managing DLQ Using tdbg
+When a task in the Temporal server encounters a terminal error, it may be enqueued in the Dead Letter Queue (DLQ). 
+To effectively manage these DLQ messages, follow these steps:
+
+1. There is a metric `dlq_writes`, which will be incremented each time a message is enqueued to the DLQ.
```

Comment:
Add: You can use this to figure out when a task encountered a terminal error and needs manual resolution.